public/data/amehi0index.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Amelia hill",
+  "type": "personal",
+  "bio": "Frontend Web Developer | CS Student",
+  "avatar": "https://github.com/amehi0index.png",
+  "links": [
+    {
+      "name": "See my Portfolio",
+      "url": "https://www.ameliahill.me",
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/amehi0index",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/amehi0index",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I think it'd look better if you capitalize the `i` here 😅 What do you think? Thanks.
```suggestion
      "name": "Follow me on LinkedIn",
```